ui/apps/platform/src/Components/FilteredWorkflowSelector/FilteredWorkflowSelector.tsx

```diff
@@ -30,7 +40,7 @@ function FilteredWorkflowSelector({
             isOpen={isSelectOpen}
             selected={filteredWorkflowState}
             onSelect={(_, value) => {
-                setFilteredWorkflowState(value);
+                onChangeFilteredWorkflowState(ensureFilteredWorkflowState(value));
```

Comment:
Although not necessarily a blocker to merge:
* help me understand the need to ensure `value` here in `Select` element that has known values
    It feels like TypeScript should be able to validate that select options correspond to workflow state.
* versus values from (untrusted) query string of page address

Reply:
I guess that is true. It may be unnecessary now that I think about it a bit more